### PR TITLE
Edits to Lasso PR5085

### PR DESCRIPTION
**Pull request by ebgitelman**
Ready for review/merge.

### CODE REVIEW

File: product_docs/docs/lasso/4/release-notes.mdx [hunk 1]
```diff
@@ -2,182 +2,161 @@
...
 
+| Type           | Description | Addresses |
+|----------------|-------------|-----------|
+| Feature        | Implemented EDB Postgres Advanced Server and 2ndQPG flavor detection. | DC-875 |
+| Feature        | Now gathering EDB Postgres Advanced Server Catalog - Views related to partitioned tables in Oracle syntax. | (DC-266) |
+| Feature        | Added support for PostgreSQL 16. | DC-887 |
+| Feature        | Added support for RHEL9. | DC-802, DC-803 |
+| Bug fix        | Fixed situation when `ls pg_data` command takes much time to complete with extremely large number of files. Added a default `timeout` of `120s` for this case. | DC-845 |
 
 ## Lasso - Version 4.10.0 (2023-11-07)
```
Comment (josh-heyer) [inline product_docs/docs/lasso/4/release-notes.mdx:17]: Per https://github.com/EnterpriseDB/docs/blob/develop/docs/agreements/release-notes-guidelines.md we should be putting the date after the heading, e.g.

```
Released: 07 Nov 2023
```

File: product_docs/docs/lasso/4/describe.mdx [hunk 2]
```diff
@@ -4,2237 +4,2092 @@ navTitle: Gathered data details
...
-no known security impact.
+**Security impact:** Low &mdash; 
+No known security impact.
 
 ### etcd CLI (`etcd_cli`)
 
-Gathers the output of some `etcdctl` commands, if `etcdctl` is
-available in the server. The commands are: endpoint status and
-endpoint health.
+Gathers the output of some etcdctl commands, if etcdctl is
```
Comment (josh-heyer) [inline product_docs/docs/lasso/4/describe.mdx:82]: Inconsistent formatting: `etcdctl` is a CLI tool with various commands, which would be invoked via the syntax `etcdctl <command>`, e.g. `etcdctl endpoint status` - most other such commands listed here are formatted as code, so this command should be as well.

Comment (josh-heyer) [inline product_docs/docs/lasso/4/describe.mdx:82]: Inconsistent formatting: `etcdctl` is a CLI tool with various commands, which would be invoked via the syntax `etcdctl <command>`, e.g. `etcdctl endpoint status` - most other such commands listed here are formatted as code, so this command should be as well.

File: product_docs/docs/lasso/4/describe.mdx [hunk 3]
```diff
@@ -4,2237 +4,2092 @@ navTitle: Gathered data details
...
-no known security impact.
+**Security impact:** low &mdash; 
+No known security impact.
 
 ### HARP CLI (`harp_cli`)
 
-Gathers output of a few `harpctl` command outputs using the
-`config.yml` file which is found in the server. The commands are:
-cluster, proxies, locations, nodes, and version.
+Gathers output of a few harpctl command outputs using the
```
Comment (josh-heyer) [inline product_docs/docs/lasso/4/describe.mdx:127]: Inconsistent formatting: `harpctl` is a CLI tool with various commands, which would be invoked via the syntax `harpctl <command>`, e.g. `harpctl version` - most other such commands listed here are formatted as code, so this command should be as well.

Note: the command itself describes this inaccurately, and this inaccuracy is echoed below in the report output; the command issued *will be* of the form `harpctl ...`, not `harp ...`

File: product_docs/docs/lasso/4/describe.mdx [hunk 4]
```diff
@@ -4,2237 +4,2092 @@ navTitle: Gathered data details
...
-SELinux status from `sestatus`
-
-Report output:
+SELinux status from `sestatus`.
 
-  * file `/linux/sestatus.data`: Output from 'setstatus'
+**Report output:**
 
-**Depth:** surface
+  * File `/linux/sestatus.data`: Output from `setstatus`
```
Comment (josh-heyer) [inline product_docs/docs/lasso/4/describe.mdx:368]: This is a typo in the original file (and the command's own documentation) - the actual command is **se**status, not se**t**status; it is correct above.

**Review by josh-heyer — DISMISSED** [review]
Looks great! Noted a handful of minor issues inline.

File: product_docs/docs/lasso/4/release-notes.mdx (hunk 1 above)
Comment (ebgitelman) [reply product_docs/docs/lasso/4/release-notes.mdx:17]: I'm going to take the date out of all of the headers and put them in the text.

File: product_docs/docs/lasso/4/describe.mdx (hunk 2 above)
Comment (ebgitelman) [reply product_docs/docs/lasso/4/describe.mdx:82]: If it's a command then tagged, if it's a tool then not tagged. Let me take a look at the source here...endpoint status and endpoint health should be tagged.

File: product_docs/docs/lasso/4/describe.mdx (hunk 4 above)
Comment (ebgitelman) [reply product_docs/docs/lasso/4/describe.mdx:368]: I fixed this in the description (output from `sestatus`

**Review by josh-heyer — DISMISSED** [review]

**Review by josh-heyer — APPROVED** [review]

**Pull request merged**